aws/lambda/clickhouse-replicator-dynamo/lambda_function.py

```diff
@@ -165,19 +166,24 @@ def upsert_document(record: Any) -> None:
         body = handle_workflow_job(body)
 
     id = extract_dynamodb_key(record)
+    print(f"Parsing {id}: {json.dumps(body)}")
```

Comment:
Do you really want to print those? And shouldn't we be using some sort of structured logger here?
```suggestion
```

Reply:
Its been surprisingly useful for debugging.

I'll switch to structured logging in a different PR, need to test what it looks like first